network_wrangler/transitnetwork.py

```diff
@@ -1241,6 +1245,257 @@ def apply_transit_managed_lane(
+
+                
+                for i in trip["headway_sec"]:
+                    # add trip
+                    trip_id = f"trip{trip_index}_{shape_id}"
+                    add_trips_dict = {
+                        "route_id": route["route_id"],
+                        "direction_id": trip["direction_id"],
+                        "trip_id": trip_id,
+                        "shape_id": shape_id,
+                        "agency_raw_name": route["agency_raw_name"]
+                    }
+                    add_trips_df = pd.DataFrame([add_trips_dict]).astype(trip_col_dtypes)
+                    trips_df = pd.concat([trips_df, add_trips_df], ignore_index=True, sort=False)
+
+                    # add frequency
+                    headway_secs = list(i.values())[0]
+                    time_range = list(i.keys())[0]
+                    time_range = [time.strip().strip("'") for time in time_range.strip("()").split(',')]
+                    start_time = parse_time_spans(time_range)[0]
+                    end_time = parse_time_spans(time_range)[1]
+
+                    add_freqs_dict = {
+                        "trip_id": trip_id,
+                        "headway_secs": headway_secs,
+                        "start_time": start_time,
+                        "end_time": end_time,
+                        "agency_raw_name": route["agency_raw_name"]
+                    }
+                    add_freqs_df = pd.DataFrame([add_freqs_dict]).astype(freq_col_dtypes)
+                    frequencies_df = pd.concat([frequencies_df, add_freqs_df], ignore_index=True, sort=False)
+
+                    # add stop and stop_times
+                    stop_model_node_id_list = []
+                    pickup_type = []
+                    drop_off_type = []
+
+                    for i in trip['routing']:
+                        if (isinstance(i, dict) and 
+                           list(i.values())[0] is not None and 
+                           list(i.values())[0].get('stop')):
+                            stop_model_node_id_list.append(list(i.keys())[0])
+                            drop_off_type.append(0 if list(i.values())[0].get('alight', True) else 1)
+                            pickup_type.append(0 if list(i.values())[0].get('board', True) else 1) 
+
+                    # used to build stop_time
+                    stop_id_list = [] 
+
+                    # used to add new stops if they are not in the stops.txt
+                    new_stop_id_list = []
+                    model_node_id_list = []
+                    stop_lat_list = []
+                    stop_lon_list = []
+
+                    for s in stop_model_node_id_list:
+                        s = int(float(s))
+                        if s in stop_id_xref_dict.keys():
+                            existing_agency_raw_name = stops_final_df[stops_final_df['model_node_id'].astype(float).astype(int)==s]['agency_raw_name'].iloc[0]
```

Comment:
Can existing `agency_raw_name` be more than one unique value? E.g., Metro and MVTA buses can use the same stop node?